ci/test/00_setup_env_native_msan.sh

```diff
@@ -20,3 +20,4 @@ export GOAL="install"
 export BITCOIN_CONFIG="--enable-wallet --with-sanitizers=memory --with-asm=no --prefix=${BASE_ROOT_DIR}/depends/x86_64-pc-linux-gnu/ CC=clang CXX=clang++ CFLAGS='${MSAN_FLAGS}' CXXFLAGS='${MSAN_AND_LIBCXX_FLAGS}' BDB_LIBS='-L${BDB_PREFIX}/lib -ldb_cxx-4.8' BDB_CFLAGS='-I${BDB_PREFIX}/include'"
 export USE_MEMORY_SANITIZER="true"
 export RUN_FUNCTIONAL_TESTS="false"
+export CCACHE_SIZE=250M
```

Comment:
Effective size is now 150MB:

https://travis-ci.org/github/bitcoin/bitcoin/jobs/708074350#L3511

Reply:
Still 150 after two runs: https://travis-ci.org/github/bitcoin/bitcoin/jobs/712518047#L3500